modules/search/class.jetpack-search.php

```diff
@@ -207,6 +208,27 @@ public function init_hooks() {
 		add_action( 'jetpack_deactivate_module_search', array( $this, 'move_search_widgets_to_inactive' ) );
 	}
 
+	/**
```

Comment:
What's the motivation for turning off the default search on the backend? Will it mean that users who aren't able to load Instant Search won't be able to use the basic search as a fallback?

Reply:
This was an attempt to address what to show behind the overlay when the user navigates directly to `/?s=seach-term-here`. In hindsight, I don't think this is a good idea; I'll revert this change today.